### ISSUE TITLE
Eliminator is broken

### ISSUE THREAD

**Issue by shnarazk** [shnarazk/splr#43]
At least for 2 weaks eliminator has been broken.

**Comment by shnarazk** [comment]
https://github.com/shnarazk/splr/blob/49d700ab0874f25b761213aed2a8746e2949c9a6/src/eliminator.rs#L353-L360

L360 is different from L359. Resolved by 82dae3750980ac0e5485e4c1e2fedd9fe4cddc40.

**Comment by shnarazk** [comment]
```
sh verify.sh
A valid assignment set for SR19/26_stack_cas_longest_true-unreach-call.i-cbmc-u2-sc2016.cnf is found in .ans_26_stack_cas_longest_true-unreach-call.i-cbmc-u2-sc2016.cnf
A valid assignment set for SR19/ASG_72_keystream76_2.cnf is found in .ans_ASG_72_keystream76_2.cnf
A valid assignment set for SR19/Bibd-sc-10-03-08_c18.cnf is found in .ans_Bibd-sc-10-03-08_c18.cnf
A valid assignment set for SR19/CarSequencing-90-01_c18.cnf is found in .ans_CarSequencing-90-01_c18.cnf
A valid assignment set for SR19/CarSequencing-90-02_c18.cnf is found in .ans_CarSequencing-90-02_c18.cnf
A valid assignment set for SR19/Eternity-10-05_c18.cnf is found in .ans_Eternity-10-05_c18.cnf
A valid assignment set for SR19/LABS_n087_goal008-sc2013.cnf is found in .ans_LABS_n087_goal008-sc2013.cnf
A valid assignment set for SR19/Mario-t-hard-2_c18.cnf is found in .ans_Mario-t-hard-2_c18.cnf
A valid assignment set for SR19/Mario-t-hard-5_c18.cnf is found in .ans_Mario-t-hard-5_c18.cnf
A valid assignment set for SR19/Mickey_out250_known_last147_0.cnf is found in .ans_Mickey_out250_known_last147_0.cnf
A valid assignment set for SR19/Subisomorphism-g10-g27_c18.cnf is found in .ans_Subisomorphism-g10-g27_c18.cnf
A valid assignment set for SR19/Tal-07_c18.cnf is found in .ans_Tal-07_c18.cnf
A valid assignment set for SR19/aaai10-planning-ipc5-pathways-17-step21-sc2011.cnf is found in .ans_aaai10-planning-ipc5-pathways-17-step21-sc2011.cnf
A valid assignment set for SR19/atco_enc1_opt1_04_32-sc2014.cnf is found in .ans_atco_enc1_opt1_04_32-sc2014.cnf
A valid assignment set for SR19/atco_enc1_opt2_10_16-sc2014.cnf is found in .ans_atco_enc1_opt2_10_16-sc2014.cnf
A valid assignment set for SR19/atco_enc2_opt2_10_21-sc2014.cnf is found in .ans_atco_enc2_opt2_10_21-sc2014.cnf
A valid assignment set for SR19/b04_s_2_unknown_pre-sc2013.cnf is found in .ans_b04_s_2_unknown_pre-sc2013.cnf
A valid assignment set for SR19/b04_s_unknown-sc2013.cnf is found in .ans_b04_s_unknown-sc2013.cnf
A valid assignment set for SR19/frb-40-19-1-mgd_c18.cnf is found in .ans_frb-40-19-1-mgd_c18.cnf
A valid assignment set for SR19/hcp_CP14_14.cnf is found in .ans_hcp_CP14_14.cnf
A valid assignment set for SR19/hcp_CP16_16.cnf is found in .ans_hcp_CP16_16.cnf
A valid assignment set for SR19/hcp_CP18_18.cnf is found in .ans_hcp_CP18_18.cnf
A valid assignment set for SR19/hcp_r20_20.cnf is found in .ans_hcp_r20_20.cnf
A valid assignment set for SR19/hcp_r26_26.cnf is found in .ans_hcp_r26_26.cnf
A valid assignment set for SR19/jgiraldezlevy.2200.9086.08.40.149-sr2015.cnf is found in .ans_jgiraldezlevy.2200.9086.08.40.149-sr2015.cnf
A valid assignment set for SR19/knight_16.cnf is found in .ans_knight_16.cnf
A valid assignment set for SR19/mod4block_2vars_10gates_u2_autoenc-sc2009.cnf is found in .ans_mod4block_2vars_10gates_u2_autoenc-sc2009.cnf
A valid assignment set for SR19/mrpp_6x6#12_24-sc2013.cnf is found in .ans_mrpp_6x6#12_24-sc2013.cnf
A valid assignment set for SR19/partial-10-11-s-sc2007.cnf is found in .ans_partial-10-11-s-sc2007.cnf
A valid assignment set for SR19/partial-5-17-s-sc2007.cnf is found in .ans_partial-5-17-s-sc2007.cnf
A valid assignment set for SR19/rbsat-v945c61409gyes9-sc2009.cnf is found in .ans_rbsat-v945c61409gyes9-sc2009.cnf
A valid assignment set for SR19/satcoin-genesis-SAT-10-sc2018.cnf is found in .ans_satcoin-genesis-SAT-10-sc2018.cnf
A valid assignment set for SR19/size_5_5_5_i171_r12.cnf is found in .ans_size_5_5_5_i171_r12.cnf
A valid assignment set for SR19/size_5_5_5_i251_r12.cnf is found in .ans_size_5_5_5_i251_r12.cnf
A valid assignment set for SR19/sv-comp19_prop-reachsafety.25_stack_longest_false-unreach-call.i-witness.cnf is found in .ans_sv-comp19_prop-reachsafety.25_stack_longest_false-unreach-call.i-witness.cnf
A valid assignment set for SR19/sv-comp19_prop-reachsafety.26_stack_cas_longer_false-unreach-call.i-witness.cnf is found in .ans_sv-comp19_prop-reachsafety.26_stack_cas_longer_false-unreach-call.i-witness.cnf
A valid assignment set for SR19/sv-comp19_prop-reachsafety.newton_3_8_false-unreach-call_true-termination.i-witness.cnf is found in .ans_sv-comp19_prop-reachsafety.newton_3_8_false-unreach-call_true-termination.i-witness.cnf
A valid assignment set for SR19/sv-comp19_prop-reachsafety.queue_longer_false-unreach-call.i-witness.cnf is found in .ans_sv-comp19_prop-reachsafety.queue_longer_false-unreach-call.i-witness.cnf
A valid assignment set for SR19/sv-comp19_prop-reachsafety.race-4_2-thread_local_vars_false-unreach-call.i-witness.cnf is found in .ans_sv-comp19_prop-reachsafety.race-4_2-thread_local_vars_false-unreach-call.i-witness.cnf
A valid assignment set for SR19/total-10-15-s-sc2007.cnf is found in .ans_total-10-15-s-sc2007.cnf
A valid assignment set for SR19/toughsat_22bits_1.cnf is found in .ans_toughsat_22bits_1.cnf
# UNSAT:0, 207, f6nidw-sc2012.cnf
s VERIFIED
s VERIFIED UNSAT

# UNSAT:1, 354, sv-comp19_prop-reachsafety.newton_2_2_true-unreach-call_true-termination.i-witness.cnf
s VERIFIED
s VERIFIED UNSAT

# UNSAT:2, 355, sv-comp19_prop-reachsafety.newton_2_3_true-unreach-call_true-termination.i-witness.cnf
s VERIFIED
s VERIFIED UNSAT

# UNSAT:3, 365, sv-comp19_prop-reachsafety.sine_8_true-unreach-call_true-termination.i-witness.cnf
s VERIFIED
s VERIFIED UNSAT

# UNSAT:4, 366, sv-comp19_prop-reachsafety.square_5_true-unreach-call_true-termination.i-witness.cnf
s VERIFIED
s VERIFIED UNSAT
$
```
